docs/smb_bio.md

```diff
@@ -0,0 +1,3 @@
+# SMB Bio
+
+- name: scott
```

Comment:
Let's view the `files changed` tab in the upper portion of the PR, view the file, click the edit icon and then commit